app/src/main/java/e/josephmolina/saywhat/Speech/PollyClient.java

```diff
@@ -0,0 +1,62 @@
+package e.josephmolina.saywhat.Speech;
+
+import android.media.AudioManager;
+import android.media.MediaPlayer;
+import android.util.Log;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.mobile.client.AWSMobileClient;
+import com.amazonaws.services.polly.AmazonPollyPresigningClient;
+import com.amazonaws.services.polly.model.OutputFormat;
+import com.amazonaws.services.polly.model.SynthesizeSpeechPresignRequest;
+
+import java.io.IOException;
+import java.net.URL;
+
+import e.josephmolina.saywhat.Utils.Utils;
+import e.josephmolina.saywhat.language.Interpret;
+
+public class PollyClient {
```

Comment:
Can we place the PollyClient inside the language package and rename the PollyClient to something like Speech?

Reply:
Yeah this can be done and will be fixed in the next commit.